### DISCUSSION TITLE
[Feature] Matomo integration

### DISCUSSION THREAD

**Discussion by oramarin** [immich-app/immich#14393] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be nice to be able to integrate [Matomo Analytics](https://matomo.org/) script ref to all pages of the server to get some infos on who is visiting the service. Maybe it is already posssible but I couldn't find ref on internet.
Really great project ! 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by Dusza** [comment]
I agree however I'd change scope so any script or HTML could be added to the HEAD or BODY tags.  Some user might have something other than Matomo that they want to implement.

**Comment by ynmhu** [comment]
> ### I have searched the existing feature requests to make sure this is not a duplicate request.
> * [x]  Yes
> 
> ### The feature
> It would be nice to be able to integrate [Matomo Analytics](https://matomo.org/) script ref to all pages of the server to get some infos on who is visiting the service. Maybe it is already posssible but I couldn't find ref on internet. Really great project !
> 
> ### Platform
> * [x]  Server
> * [ ]  Web
> * [ ]  Mobile

https://forum.ynm.hu/t/immich-matomo-integration/290 I made for me so surly you can add your site to :)

**Comment by bo0tzz** [comment]
Closing in favour of #19444
